tests/functional/test_freeze.py

```diff
@@ -134,23 +122,14 @@ def test_freeze_git_clone(script, tmpdir):
         """
             Script result: ...pip freeze
             -- stdout: --------------------
-            ...-e %s@...#egg=pip_test_package-...
+            ...-e git+...#egg=version_pkg-master
```

Comment:
Oh I was tempted too with the same thought. I think that would be nice but I'm leaning towards leaving that for another PR. I like to keep PRs small and simple and to try to get then merged quickly so they don't get overlooked or become stale. Incremental progress, agile, yada yada, etc. :)

I'll do it if course if it's required, but you sound open to deferring it.